device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/InternalClient.java

```diff
@@ -980,4 +1018,13 @@ private void commonConstructorVerification(IotHubConnectionString connectionStri
             throw new UnsupportedOperationException("Communication with edgehub only supported by MQTT/MQTT_WS and AMQPS/AMQPS_WS");
         }
     }
+
+    private void verifyNotUnregisteredIfMultiplexing()
+    {
+        // deviceIO is only ever null when a client was registered to a multiplexing client, became unregistered, and hasn't be re-registered yet.
```

Comment:
is this the only indication of the client using multiplexing? no boolean indicator anywhere? it would probably be useful to have.

Reply:
We have an enum ```ioTHubConnectionType``` at the DeviceClient level, but not at the internal client level. I'm on the fence about adding one since ModuleClient also uses this class and has no concept of multiplexing